packages/cashier/src/pages/payment-agent/payment-agent-container/payment-agent-container.tsx

```diff
@@ -78,14 +80,18 @@ const PaymentAgentContainer = observer(({ is_deposit }: TPaymentAgentContainer)
 
     return (
         <React.Fragment>
-            {!has_payment_agent_search_warning && (
-                <SideNote className='payment-agent-list__side-note' has_title={false} is_mobile>
-                    <PaymentAgentDisclaimer />
-                </SideNote>
+            {!isDesktop && (
```

Comment:
@nijil-deriv 
question.
is it possible to pass is_mobile={!isDesktop} prop to `SideNote` instead of doing condition rendering?